<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>256</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_kolvo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Кол-во сторок:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_kolvo">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Заполнить случайно</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tabl">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>принять</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sum_nech">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>сумма нечетных</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sumnech">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sum_ch">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>сумма четных</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sumch">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_polog_100">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Кол-во положительных элементов &lt;100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_polog_100">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton__proiz_polog">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Произведение положительных</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_proiz_polog">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>250</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_kolvo_otr_zn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Кол-во отрицательных элементов в массиве</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_kolvo_otr_zn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cht_neind">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Подсчет элементов с нечетными индексами</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cht_ind">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Подсчет элементов с четными индексами</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cht_ind">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>370</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_cht_neind">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>420</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_factorial">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>490</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_factorial">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>490</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Самое частое значение</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1258</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
